md/dev/mediquette.txt

```diff
@@ -0,0 +1,74 @@
+
+Etiquette @medic
+
+Be kind to the plants and animals.
+
+Slack
+
+  1) Try to pay attention what channel you are writing in and keep noise
+     (tangential or unrelated discussion) to a minimum.  Use threads when
+     appropriate to keep tangential discussion contained and out of the main
+     channel.   Each channel should have a purpose with a description that
+     explains appropriate usage.
+
+  2) Never delete or edit someone else's content.  If you think something is
+     inappropriate then notify the person first and ask them politely to remove
+     it.
+
+  3) In general deleting something you already posted is poor etiquette.
+     Because it changes the context of the discussion.  If you have a problem
+     with something you wrote then just create a thread on it or just repost it
+     with your edits.
+
+
+Git
+
+ Commit Titles
+
+    The commit title is a short accurate summary about what code was changed.
+    Someone should be able to read the title of the commit message and have an
+    idea of what part of the code has changed and the impact it might have on
+    things. Use words like: "bugfix", "refactored", "updated" and "fixed" .
+    Use the past tense to signify that the change is done and not something
+    still happening currently or in the future.
+
+    Bad:
+      Fix Typo
+
+    Good: 
+      Fixed typo in xform upload script.
+
+ Commit Bodies
```

Comment:
We've had contrasting conversations about this one, with no authoritative resolution.

What you have written roughly aligned with my beliefs / feelings. However, others believe GH tickets are the single source of truth, and others again think actual raw commited code is.

I don't know if we want to re-hash it, because it feels like a tabs vs. spaces level discussion, that will go nowhere slowly. When we last discussed it I personally resolved to do what I thought was best, and hopefully if my methods are more valid that will become apparent over time (and vice versa).